src/time_machine/__init__.py

```diff
@@ -117,6 +118,12 @@ def extract_timestamp_tzname(
         if dest.tzinfo is None:
             dest = dest.replace(tzinfo=dt.timezone.utc)
         timestamp = dest.timestamp()
+    elif isinstance(dest, dt.timedelta):
+        if escape_hatch.is_travelling():
```

Comment:
No need to use the escape hatch API internally
```suggestion
        if coordinates_stack:
```